src/client/proxy/socks.cc

```diff
@@ -49,606 +49,725 @@
 namespace kovri {
 namespace client {
 
-static const std::size_t SOCKS_BUFFER_SIZE = 8192;
+static const std::size_t SOCKSBufferSize = 8192;
 // Limit for socks5 and bad idea to traverse
-static const std::size_t MAX_SOCKS_HOSTNAME_SIZE = 255;
+static const std::size_t MaxSOCKSHostSize = 255;
 
-struct SOCKSDnsAddress {
+struct SOCKSDNSAddress {
   std::uint8_t size;
-  char value[MAX_SOCKS_HOSTNAME_SIZE];
+  char value[MaxSOCKSHostSize];
   void FromString(std::string str) {
     size = str.length();
-    if (str.length() > MAX_SOCKS_HOSTNAME_SIZE)
-      size = MAX_SOCKS_HOSTNAME_SIZE;
+    if (str.length() > MaxSOCKSHostSize)
+      size = MaxSOCKSHostSize;
     memcpy(value, str.c_str(), size);
   }
-  std::string ToString() { return std::string(value, size); }
-  void push_back(char c) { value[size++] = c; }
```

Comment:
Yes, what a terrible idea it was to give the illusion of an overloaded `push_back` member; alla STL. Very confusing.